airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -185,7 +184,8 @@ def _parse_and_handle_errors(response) -> Union[MutableMapping[str, Any], List[M
             message = response.json().get("message")
 
         if response.status_code == HTTPStatus.FORBIDDEN:
-            raise HubspotAccessDenied(message, response=response)
+            """ Once hit the forbidden endpoint, we return the error message from response. """
+            return response.json()
         elif response.status_code in (HTTPStatus.UNAUTHORIZED, CLOUDFLARE_ORIGIN_DNS_ERROR):
```

Comment:
response.json() is used in several parts of this function. Maybe will we execute it at once?

Reply:
Done, we `pass` once we hit `response.status_code == HTTPStatus.FORBIDDEN:` and will return the status message when we `return response.json()` in the end of the function. 
Thanks.